evap/evaluation/tests/test_models_logging.py

```diff
@@ -0,0 +1,88 @@
+from datetime import date, datetime, timedelta
+
+from django.test import TestCase
+from django.utils.formats import localize
+from model_bakery import baker
+
+from evap.evaluation.models import Evaluation, Course, Contribution, Questionnaire
+from evap.evaluation.models_logging import FieldAction
+
+
+class TestLoggedModel(TestCase):
+    def setUp(self):
+        self.old_start_date = datetime.now()
+        self.new_start_date = self.old_start_date + timedelta(days=10)
+
+        self.course = baker.make(Course)
+        self.course.save()
```

Comment:
I think you don't need that, baker.make will create the object and store the row in the database (happens more often)